src/Microsoft.Azure.WebJobs/AutoResolveAttribute.cs

```diff
@@ -25,5 +26,10 @@ public AutoResolveAttribute()
         /// in the Microsoft.Azure.WebJobs.Host assembly.
         /// </summary>
         public Type ResolutionPolicyType { get; set; }
+
+        /// <summary>
+        /// If this is empty, then specify a default. 
```

Comment:
nit: "Gets or sets the value to use if the target property doesn't contain a value."?